index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Fork Me! FCC: Test Suite Template</title>
+  <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+
+  <div>
+    <header id="header">
+      <h2> Handcrafted, home-made masterpieces </h2>
+      <div>
+        <img id="header-img"
+          src="https://cdn.freecodecamp.org/testable-projects-fcc/images/product-landing-page-logo.png"
+          alt="original trombones logo">
+      </div>
+
+      <div>
+        <nav id="nav-bar">
+          <a href="#How-It-Works" class="nav-link">
+            How It Works</a>
+          <a href="#Feature" class="nav-link"> Feature</a>
+          <a href="#Pricing" class="nav-link"> Pricing</a>
+        </nav>
+      </div>
+    </header>
+    <div>
+      <iframe id="video" height="315"
+        src="https://www.youtube-nocookie.com/embed/y8Yv4pnO7qc?rel=0&amp;controls=0&amp;showinfo=0" frameborder="0"
+        allowfullscreen=""></iframe>
+    </div>
+    <div>
+      <form action="https://www.freecodecamp.com/email-submit" id="form">
+        <input type="email" name="email" id="email" placeholder="Please enter your email" validate>
+        <input type="submit" value="Submit" id="submit">
+      </form>
+    </div>
+  </div>
+  <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+  <script src="js/index.js"></script>
+</body>
+
+</html>
```

Comment:
Kindly practice indentation whenever you're working on your project for this will help others to read your code and it will look clean.

Reply:
- [ ] Good job on this comment too but please be more specific because they can't understand it sometimes.